docs/decisions/0008-additional-redemption-metadata.rst

```diff
@@ -0,0 +1,55 @@
+0008 Additional Metadata for Policy Redemption
+************************************************************
+
+Status
+======
+
+Proposed
+
+Context
+=======
+
+This document intends to outline modifications to existing API endpoint(s) such that MFEs can pass additional metadata into the redemption flow. This extension was devised to support required GEAG meatadata such as DOB and terms acceptance dates.
+
+Decision
+=======
+
+The GEAG system requires additional metadata about a learner in order to process an allocation. This additional metadata (name, dob, etc) are collected during the enrollment flow on the edX side before enrollment. This additional metadata is not persisted anywhere on the edX side. Because of the lack of persistence and the placement of this information collection on the edX side of the process we decided to create a facility for the frontend to pass along additional metadata into the redemption flow.
```

Comment:
> Because of the lack of persistence...

[clarification] Is part of the decision here to start persisting the metadata (likely alongside the transaction I'd assume)? Might be worth expanding on the decision to explicitly mention that, if so.

Reply:
By lack of persistence i meant that the MFE isn't saving it anywhere - such as the user profile - which would allow the subsidy system to retrieve it for enrollment into GEAG. Because its scope of existence is only this enrollment flow we'll need to pass it directly into the redemption flow. TBD on how much of this information will be persisted by the Subsidy service.